frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml

```diff
@@ -57,128 +57,102 @@
     - summary
 
 -
-  name: Shortlist criteria
+  name: Shortlist and evaluation process
   editable: True
   step: 2
   description: |
-    After you’ve published your requirements, suppliers will apply and tell you:
+    There are 3 stages to finding the supplier that best meets your needs.
 
-    - when they can start work
-    - if they have the essential skills and experience
-    - which nice-to-have skills and experience they have
+    **1\. View suppliers who applied**<br />
```

Comment:
As a note, you can make markdown output `<br>` tags by adding two spaces to the end of the line you need it after. I replaced all those we had in this pull request in 7145b55